services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/ApplicationPdfService.kt

```diff
@@ -0,0 +1,184 @@
+package fi.hel.haitaton.hanke.allu
+
+import com.lowagie.text.Chunk
+import com.lowagie.text.Document
+import com.lowagie.text.Element
+import com.lowagie.text.Font
+import com.lowagie.text.PageSize
+import com.lowagie.text.Paragraph
+import com.lowagie.text.Phrase
+import com.lowagie.text.Rectangle
+import com.lowagie.text.pdf.PdfPTable
+import com.lowagie.text.pdf.PdfWriter
+import fi.hel.haitaton.hanke.application.CableReportApplicationData
+import fi.hel.haitaton.hanke.application.Contact
+import fi.hel.haitaton.hanke.application.CustomerWithContacts
+import fi.hel.haitaton.hanke.application.PostalAddress
+import java.io.ByteArrayOutputStream
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import org.springframework.stereotype.Service
+
+/**
+ * Transform an application to a PDF. The PDF is added as an attachment when sending the application
+ * to Allu. It will be archived along the application and decision to show how the user inputted the
+ * application in Haitaton, as the data models of Haitaton and Allu differ slightly.
+ */
+@Service
+object ApplicationPdfService {
```

Comment:
It seems that other spring annotated services are classes. Is there a reason why class is not used here?

Reply:
Thank you for noticing. This isn't supposed to be `@Service` anymore. It was at some point during development, but I refactored this so that it has no dependencies, so it can be an object.